tests/cases/standalone/common/tql/basic.result

```diff
@@ -84,14 +84,6 @@ TQL EVAL (0, 10, '1s', '2s') test{k="a"};
 +-----+---------------------+---+
 | 2.0 | 1970-01-01T00:00:01 | a |
 | 2.0 | 1970-01-01T00:00:02 | a |
-| 2.0 | 1970-01-01T00:00:03 | a |
```

Comment:
I found the following comment in L79, is it outdated?
```
-- 'lookback' parameter is not fully supported, the test has to be updatedTQL EVAL (0, 10, '1s', '2s') test{k="a"};
```

Reply:
good spot! Thanks @evenyag 
I've actually put it exactly to drop when the test got updated but miserably failed to do it 😄